src/utils.ts

```diff
@@ -0,0 +1,90 @@
+import * as vscode from 'vscode';
+
+export async function runJar(jarname: string) {
+    // Find .jar files recursively in each workspace and in all directories
```

Comment:
```suggestion
    // Inform the user that the extension is attempting to find the jar file
    vscode.window.showInformationMessage(`Attempting to find ${jarname}.jar...`);
    // Find .jar files recursively in each workspace and in all directories
```

We should add this line since finding the jar files can take quite some time, especially in larger workspaces with more folders.